plugins/test/noop/debug/debug.go

```diff
@@ -70,3 +72,25 @@ func (debug *Debug) WriteDebug(debugFilePath string) error {
 
 	return nil
 }
+
+// WriteCommandLog appends the executed cni command to the record file
+func WriteCommandLog(path string, cmd string) error {
+	fp, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0o655)
```

Comment:
Why permission is `655`, `rw-r-xr-x`? I'm just curious why 'x' is required to group/others?